packages/manager/src/features/CloudPulse/shared/CloudPulseCustomSelect.tsx

```diff
@@ -237,6 +243,11 @@ export const CloudPulseCustomSelect = React.memo(
             placement: 'bottom',
           },
         }}
+        textFieldProps={{
+          InputProps: {
+            required,
```

Comment:
I understood now, but can we make something isOptional and highlight only the optional filters, i feel putting required in every filter is not good.